entrypoint.py

```diff
@@ -1,19 +1,24 @@
-#!/usr/bin/python3
+#!/usr/bin/env python3
 
-from sys import argv
+from sys import argv, exit
+import re
+
+import docker
 
-from docker import Client
 
 class ImageNotFound(Exception):
     pass
 
-class MainObj(object):
+
+class Main:
     def __init__(self):
-        super(MainObj, self).__init__()
+        super(Main, self).__init__()
```

Comment:
```suggestion
        super().__init__()
```

Heh, Python has changed a lot since I wrote this :)

Reply:
Yes, I didn't notice this change.

Reply:
Because `Main` is not extend any explicit base class, so `super().__init__()` is optional and recommend to ignore it IMO.